deployers/servicedeployer.go

```diff
@@ -1088,8 +1088,14 @@ func (deployer *ServiceDeployer) createSwaggerApi(api *whisk.ApiCreateRequest) e
 	var response *http.Response
 
 	apiCreateReqOptions := deployer.Deployment.SwaggerApiOptions
-	apiCreateReqOptions.SpaceGuid = strings.Split(deployer.Client.Config.AuthToken, ":")[0]
 	apiCreateReqOptions.AccessToken = deployer.Client.Config.ApigwAccessToken
+	// In the case of IAM namespaces, we must use the ApigwTenantId as the SpaceGuid
```

Comment:
Applying the same IAM token (securekey) logic here as we have added elsewhere... LGTM